src/reconcile/rec_visibility.c

```diff
@@ -790,7 +796,7 @@ __rec_upd_select(WT_SESSION_IMPL *session, WTI_RECONCILE *r, WT_CELL_UNPACK_KV *
                      * If we have seen a tombstone that rolled back the prepared update, delete the
                      * key from the disk.
                      */
-                    if (prepare_rollback_tombstone != NULL)
+                    if (prepare_rollback_tombstone != NULL) // do we need to break here
```

Comment:
For in-memory case, we should not have any prepare rollback tombstone. We need to change the prepared rollback logic to ensure that as well. And here we don't need to break.